### PR TITLE
A LOT OF CHANGES 

### PR DESCRIPTION
**Pull request by oberruti**

### CODE REVIEW

File: practico_08/autogestion_alumnos/flask_backend/main/routes.py [hunk 1]
```diff
@@ -5,18 +5,18 @@
...
+#
+# @main.route('/<default_page>')
+# def redirected_home(default_page):
+#     return go_to("users.log_in")
+#
+#
+# @main.route('/dashboard')
+# @login_required
+# def dashboard():
+#     return render_template('dashboard.html')
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/flask_backend/main/routes.py:22]: We should delete this

File: practico_08/autogestion_alumnos/react-frontend/src/components/navigation/components/menuEntry.tsx [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+            padding: '0 16px',
+        },
+        triangle: {
+            border: 'none',
+            backgroundColor: Colors.background.menu3,
+        },
+    }
+    return (
+        <div style={styles.container}>
+            {/*<TooltipTriangle*/}
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/react-frontend/src/components/navigation/components/menuEntry.tsx:37]: probably we want to delete this

File: practico_08/autogestion_alumnos/react-frontend/src/components/navigation/components/menuEntry.tsx [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+        disabledTrigger: {
+            cursor: 'default',
+        },
+    }
+    const triggerStyle = props.isDisabled ? styles.disabledTrigger : null
+    const tooltip = <EntryTooltip tooltip={props.tooltip} />
+    return (
+        <HorizontalStack style={styles.stack}>
+            <EntryBorder isHighlighted={props.isActive} />
+            {/*<OnHoverTooltip*/}
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/react-frontend/src/components/navigation/components/menuEntry.tsx:109]: idem this

File: practico_08/autogestion_alumnos/react-frontend/src/components/navigation/components/menuEntry.tsx [hunk 4]
```diff
@@ -0,0 +1,245 @@
...
+            this.props.isActive,
+            this.props.isExpanded,
+            this.props.isDisabled
+        )
+        const style: Style = {
+            fontSize: 22,
+            ...triggerIconStyle,
+            ...this.props.style,
+        }
+        // return <Icon icon={this.props.icon} style={style} />
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/react-frontend/src/components/navigation/components/menuEntry.tsx:213]: we should add this again with Google icons

File: practico_08/autogestion_alumnos/react-frontend/src/components/navigation/components/singleLinkMenu.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    const logo = {
+        width: 28,
+        height: 28,
+        marginBottom: 2,
+        display: 'block',
+        outline: 'none',
+    }
+    return (
+        <Link to={'/app/dashboard/'} style={logo}>
+            <svg /*logo*/ />
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/react-frontend/src/components/navigation/components/singleLinkMenu.tsx:34]: WE SHOULD IMPLEMENT A LOGO FOR OUR APP HERE

File: practico_08/autogestion_alumnos/react-frontend/src/routes.tsx [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+                    path="/acc/login"
+                    render={() => <Login cookies={props.cookies} />}
+                />
+                <Route path="/acc/signup" component={signUp} />
+                <Route path="/acc/password-reset" component={PasswordReset} />
+            </Switch>
+        )
+}
+
+const DashboardLoader = (props: { cookies: Cookies }): JSX.Element => {
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/react-frontend/src/routes.tsx:84]: MOVE THIS TO HIS OWN MODULE

File: practico_08/autogestion_alumnos/react-frontend/src/style.ts [hunk 7]
```diff
@@ -0,0 +1,248 @@
...
+        table: '#D4DADC',
+        white: '#FFFFFF',
+        graphGrey: '#B6BEC2',
+        underline: '#69cef5',
+        darkUnderline: '#F2F4F5',
+        dark1: '#21323B', // used to separate logo from entries in the menu
+        active: '#00B9A1', // used in borders of active menu/sub-menu entries
+        selectedCellBorder: '#0E4049',
+    },
+    retain: {
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/react-frontend/src/style.ts:137]: delete this

File: practico_08/autogestion_alumnos/react-frontend/src/style.ts [hunk 8]
```diff
@@ -0,0 +1,248 @@
...
+        churnPrevention: {
+            effectiveness: '#E0D4E3',
+            switch: '#A76DB2',
+        },
+        disabled: {
+            effectiveness: '#DEE5E6',
+            switch: '#E8EEF0',
+        },
+    },
+    primary: {
```
Comment (oberruti) [inline practico_08/autogestion_alumnos/react-frontend/src/style.ts:151]: change this to our primary colors

**Review by oberruti — COMMENTED** [review]
@giovamarr please check this comments and modify them in the next PR! Thanks!!!

File: practico_08/autogestion_alumnos/flask_backend/main/routes.py (hunk 1 above)
Comment (giovamarr) [reply practico_08/autogestion_alumnos/flask_backend/main/routes.py:22]: done

**Pull request merged**